contracts/Core/Parameters.sol

```diff
@@ -23,21 +24,22 @@ contract Parameters is IParameters, ACL {
     uint256 public override aggregationRange = 3;
     uint256 public override withdrawReleasePeriod = 5;
     uint256 public override resetLockPenalty = 1;
+    uint256 public override maxAge = 100 * 10000;
```

Comment:
Why 100 * 10000?

Reply:
Just for readability. 100 epochs. 10000 per epoch.